### PR TITLE
doc updates for release alpha.16

### PR DESCRIPTION
**Pull request by daxian-dbw**
doc updates for release alpha.16

### CODE REVIEW

**Comment by iSazonov** [comment]
@daxian-dbw Travis CI again is failed on Class parser tests.
The problem always happens after [this point ](https://travis-ci.org/PowerShell/PowerShell/jobs/202106522), isn't it? 
If so then the problem is in [Get-RuntimeError](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Language/LanguageTestSupport.psm1#L22)
And there is only `[scriptblock]::Create` - Maybe we have a bug in the C# code?

**Review by TravisEz13 — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@iSazonov This is annoying and we have trouble reproducing it locally. It could be a .NET Core bug. We are investigating to generate dump in case of a hang in CI build.

**Comment by iSazonov** [comment]
Today I tried unsuccessfully to reproduce this issue on WSL. 😕  We do not make any progress without the dump.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
 Changelog
 =========
 
-v6.0.0-alpha.15 - 2016-01-18
+v6.0.0-alpha.16 - 2017-02-15
```
Comment (joeyaiello) [inline CHANGELOG.md]: I don't consider this critical for this release (don't want to block on something like this), but in the future, can we trail each line item with an Issue # where applicable? I'd like for people to be able to trackback to context for each line item.

Comment (daxian-dbw) [inline CHANGELOG.md]: That sounds a right thing to do. We can do that in future releases.

**Comment by iSazonov** [comment]
@daxian-dbw I attached a script to get PR list:
```
$a=Search-PR ([datetime]::Today.AddDays(-5)) ([datetime]::Today.AddDays(-3))
$a | Format-PRforNotes

title                                                 number user         closed_at
-----                                                 ------ ----         ---------
Fix reference to sshd service restarting                3138 SteveL-MSFT  2017-02-13T22:13:13Z
Fix class parsing (issue 3083)                          3103 iSazonov     2017-02-14T00:45:01Z
Approve console cmdlets tests                           3101 iSazonov     2017-02-14T19:00:04Z
Implement RoleDefinitions RoleCapabilityFiles keyword   3067 PaulHigin    2017-02-14T18:42:18Z
Treat unbound command parameters as command arguments   2919 AndrewGaspar 2017-02-14T00:02:57Z
``` 
[SearchPR.txt](https://github.com/PowerShell/PowerShell/files/782846/SearchPR.txt)

**Comment by daxian-dbw** [comment]
Thanks @iSazonov 😄

**Pull request merged**